heidegger_index/views.py

```diff
@@ -113,3 +113,8 @@ class URNRedirectView(LemmaDetailView):
     def get(self, *args, **kwargs):
         lemma = get_object_or_404(Lemma, urn=kwargs['urn'])
         return redirect("index:lemma-detail", slug=lemma.slug)
+
+class GNDRedirectView(LemmaDetailView):
```

Comment:
Dit heb je ook gedaan bij die URN's. Waar gebruik je deze redirect views eigenlijk voor?

Reply:
De redirects zijn er om de lemmata op te vragen vanaf de URN's/GND ID's, en anders een 404 op te werpen. Of denk jij dat ik hem direct vanaf de lemma view kan laten redirecten?